style/swift/sample.swift

```diff
@@ -85,3 +85,13 @@ struct DopeObject {
     return DopeObject(json: json)
   }
 }
+
+// Break up long function calls and initializers
+let foo = Foo(
+  bar: Bar(),
+  baz: Baz(),
+  quz: Quz(
+    x: 42,
+    y: 365
+  )
```

Comment:
I'm wondering if this example, while illustrative, might come across as a guideline in itself that you should split up arguments onto new lines even if they are short.

What about making `Quz` take more arguments so it matches up with the rule of breaking up arguments, or removing it completely?